src/phoenix.js

```diff
@@ -65,6 +65,10 @@ Vue.use(VueMeta, {
 Vue.component('drag', Drag)
 Vue.component('drop', Drop)
 
+// --- Event Bus ----
+
+Vue.prototype.$bus = new Vue()
```

Comment:
@LukasHirt can you confirm that this makes sense ?

Reply:
TBH I'd rather see event bus avoided. @marcus-herrmann is it necessary here? Would it be much trouble covering this with state? If needed to stay, pls make the bus a constant and import it only where it's needed.

Reply:
Covering it with state was my first approach, I talked with @PVince81 about that and we agreed that this approach was overengeenering/not what the store was meant to be. Keep in mind that there would be more occurances for this, to send for example focus across the interface/components that are not close to each other.

Reply:
@LukasHirt Here's the first approach mentioned above: https://github.com/owncloud/phoenix/pull/2624

Reply:
@LukasHirt are we ok with the bus then ?

Reply:
now thinking, I remember seeing that it's possible to access the root component using `this.$root`, would it be possible to use that component as the bus instead ?

Reply:
This appears to work as well. PR updated